tech.json

```diff
@@ -1808,33 +1809,32 @@
               "name": "canXRayCeilingClip",
               "techRequires": [
                 "canCeilingClip",
-                "canUseEnemies"
+                "canUseEnemies",
```

Comment:
There shouldn't be a `canUseEnemies` requirement here since there are a number of places where you can unmorph in a two tile gap ... was it brought over with the trickyIceClip refactor?

Reply:
Or maybe this tech is more enemy oriented than the name suggests

Reply:
I was talking to Maddo about this, and we could move `canUseEnemies` into the `otherRequires` and still be able to reference it without. Alternatively I was thinking we could keep `canUseEnemies` off of it and add it on with a helper, such as `h_canXRayMorphIceClip` or something?

Reply:
`canXRayCeilingClip` has 2 uses: on top of enemies and between solid tiles like in Beta PBs.  Then another one is at the top of an xray climb as seen in Bowling to avoid taking spike damage, but that is less the focus of the strat.

Helpers for both sounds good to me.  And this is used for snail clips as well as ice clips.